### DISCUSSION TITLE
Resign as maintainer

### DISCUSSION THREAD

**Discussion by martyspiewak** [paketo-buildpacks/feedback#111] [Dependencies Team]
Hi, I would like to resign as a member of the Dependencies Maintainers team as I have not been actively involved in the project for a few months now.

@paketo-buildpacks/dependencies-maintainers can I get a 👍 / 👎 ?

**Comment by martyspiewak** [comment]
@paketo-buildpacks/steering-committee Looks like I have the required votes. Could I be removed from maintainers?

**Comment by dmikusa** [comment]
Yes, I can take care of this tomorrow. Thanks for all of your help on the team!

**Comment by dmikusa** [comment]
Done

**Comment by martyspiewak** [comment]
Thanks!
